bin/pull-harmony-images

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# pull down docker images that are needed to run a basic harmony service-example reqeust
+all_images=( "harmonyservices/service-example:latest" "ghcr.io/nasa/harmony-netcdf-to-zarr:latest" )
```

Comment:
I added this to avoid harmony-in-a-box constantly trying to pull down the netcdf-to-zarr image because we have netcdf-to-zarr as one of the LOCALLY_DEPLOYED_SERVICES. Don't know the history of this. Should we configure LOCALLY_DEPLOYED_SERVICES to be only service-example?

Reply:
Yeah our default should only be harmony-service-example.